app/javascript/components/SearchResultsLayout.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { Header } from './Header/Header';
+import { Facets } from './Facets/Facets';
+import { SearchBar } from './SearchBar/SearchBar';
+import { Results } from './Results/Results';
+import { Footer } from './Footer/Footer';
+import { Identifier } from './Identifier/Identifier';
+
+interface SearchResultsLayoutProps {
+  results: any;
```

Comment:
I think we can update these props to have more specific types instead of `any`.

Reply:
Yes, I agree. As a part of Layouts, I didn't touch it. Can you confirm if the `results` will always be an array? If yes, I can use `results:[];`

Reply:
`results` should always be an array.

Reply:
@ddzz changes are ready to be reviewed.